index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+
```

Comment:
Please avoid using blank spaces in your code.

Reply:
- [ ] When giving a review, it is usually good to always includes an explanation ("Why") for each requested change so that the person whose code is being reviewed will understand why the change was necessary and why it should be implemented 😄